composer.json

```diff
@@ -21,7 +21,7 @@
     },
     "require-dev": {
         "mockery/mockery": "^1.0",
-        "phpunit/phpunit": "^7.0",
+        "phpunit/phpunit": "^9.4",
```

Comment:
Does 9.4 still work with 7.3? If not we should allow both?

Reply:
Yes it does! If not, the GH actions step would have failed already ;)